e2e-tests/Cargo.toml

```diff
@@ -42,6 +42,5 @@ name = "canister_info"
 path = "canisters/canister_info.rs"
 
 [dev-dependencies]
-candid = "0.8"
 hex = "0.4.3"
-ic-test-state-machine-client = "1"
+ic-test-state-machine-client = { git = "https://github.com/lwshang/test-state-machine-client", branch = "lwshang/candid_0.9" }
```

Comment:
What needs to happen here?

Reply:
`ic-test-state-machine-client` master also depends on `ic-cdk` which causes circular dependency.
I refactored that crate to not depend on `ic-cdk`.

The only maintainer of the crate is on vacation. It will take a while to get the [PR](https://github.com/dfinity/test-state-machine-client/pull/19) merged and release new versions.

Since this is a dev-dependency for e2e-tests, it's OK to use my patched version for now.

I'll add a TODO comment with link of the PR.